files/sidebars/webassemblysidebar.yaml

```
@@ -68,6 +142,7 @@ l10n:
   en-US:
     WebAssembly_concepts: WebAssembly concepts
     Compiling_to_Wasm: Compiling from C/C++ to WebAssembly
+    Existing_C_to_Wasm: Compiling an existing C module to WebAssembly
     Compiling_Rust_to_Wasm: Compiling from Rust to WebAssembly
     JavaScript_API: Using the WebAssembly JavaScript API
     Text_format: Understanding WebAssembly text format
```

we need an entry for:
```suggestion
    Text_format: Understanding WebAssembly text format
    Text_format_to_Wasm: Converting text format to webAssembly
```
The page title on https://developer.mozilla.org/en-US/docs/WebAssembly/Guides/Text_format_to_Wasm can also be changed

---

Are you suggesting we should change the page title? The current title doesn't look too bad for me.

---

Yes, I was suggesting we update both the page title as well as the sidebar entry - mainly to replace "Wasm" with "WebAssembly".
This is the only sidebar entry that uses "Wasm" instead of "WebAssembly", and it is pulling this directly from the page title "Converting WebAssembly text format to Wasm"